vcr/patch.py

```diff
@@ -184,13 +189,27 @@ def _requests(self):
             return ()
         return self._urllib3_patchers(cpool, requests_stubs)
 
+    @_build_patchers_from_mock_triples_decorator
     def _boto3(self):
+
         try:
-            import botocore.vendored.requests.packages.urllib3.connectionpool as cpool
+            # botocore using awsrequest
+            import botocore.awsrequest as cpool
         except ImportError:  # pragma: no cover
-            return ()
-        from .stubs import boto3_stubs
-        return self._urllib3_patchers(cpool, boto3_stubs)
+            try:
+                # botocore using vendored requests
+                import botocore.vendored.requests.packages.urllib3.connectionpool as cpool
+            except ImportError:  # pragma: no cover
+                pass
+            else:
+                from .stubs import boto3_stubs
+                yield self._urllib3_patchers(cpool, boto3_stubs)
+        else:
+            from .stubs import boto3_stubs
+            log.debug("Patching boto3 cpool with %s", cpool)
+            print('pipip')
```

Comment:
cleanup?

Reply:
In progress ;)
I might be able to push at the end of the week or so